src/test/java/org/folio/entlinks/service/links/LinksSuggestionsServiceTest.java

```diff
@@ -275,14 +307,14 @@ void shouldNotFillErrorDetailsWithDisabledAutoLinking_whenNoSubfield() {
     assertNull(field.getLinkDetails());
   }
 
-  private AuthorityParsedContent getAuthorityParsedRecordContent(String authorityField) {
-    return getAuthorityParsedRecordContent(authorityField, Map.of("a", List.of("test")));
+  private AuthorityParsedContent getAuthorityParsedRecordContent(UUID authorityId, String authorityField) {
+    return getAuthorityParsedRecordContent(authorityId, authorityField, Map.of("a", List.of("test")));
   }
 
-  private AuthorityParsedContent getAuthorityParsedRecordContent(String authorityField,
+  private AuthorityParsedContent getAuthorityParsedRecordContent(UUID authorityId, String authorityField,
```

Comment:
Instead of making each test pass authority Id - you can create additional method which will accept only "authorityField", "subfields" and will just call this method with additional AUTHORITY_ID